diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/comments/CommentedCodeTest.kt

```diff
@@ -293,4 +290,58 @@ class CommentedCodeTest : LintTestBase(::CommentsRule) {
             |   */
             """.trimMargin())
     }
+
+    @Test
+    @Tag(WarningNames.COMMENTED_OUT_CODE)
+    fun `should not trigger on Copyright and another comment`() {
+        lintMethod(
+            """
+            /*
+                Copyright (c) Your Company Name Here. 2010-2021
+            */
+            
+            package org.cqfn.diktat
+            
+            /*
+                x = 2 + 4 + 1
+            */
+            // x = 2+4
+            
+            // if true make this
+            
+            /*
+                class A {
+                
+                fun foo()  
+                
+                }
+            
+            */
+            """.trimMargin(),
+            LintError(7, 13, ruleId, "${COMMENTED_OUT_CODE.warnText()} ", false),
+            LintError(10, 13, ruleId, "${COMMENTED_OUT_CODE.warnText()} x = 2+4", false),
+            LintError(14, 13, ruleId, "${COMMENTED_OUT_CODE.warnText()} ", false)
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.COMMENTED_OUT_CODE)
+    fun `should not trigger with suppress`() {
```

Comment:
Why do you need to check suppress here?

Reply:
Because, as I added new logic for `Suppress`

Reply:
Ok